plans_schema.json

```diff
@@ -12,11 +12,11 @@
                 "formulary_url": {"type": "string", "format": "uri"},
                 "plan_contact": {"type": "string", "format": "email"},
                 "network": {"type": "array", "items": {"$ref": "#/definitions/network"}},
-                "formulary": {"type": "array", "items": {"$ref": "#/definitions/formulary"}},
```

Comment:
This was communicated by issuers and confirmed by CMS, assuming I understand it properly.